tests/e2e/assistant.spec.js

```diff
@@ -29,8 +38,1269 @@ const MediaServices = {
+    },
+    extractedURLAnalyses: {
+        "https://x.com/BreitbartNews": LinkResult.mention,
+        "https://twitter.com/pamkeyNEN": null,
+    },
+    credibilitySignals: {
+      topic: ["Law and Justice System", "Politics"],
+      genre: ["Satire"], // Questionable...
+      persuasionTechniques: [
+        "Appeal to Authority",
+        "Doubt",
+        "Repetition"
+      ],
+      subjectivity: ["None detected"]
+    }
+  },
+  // Positive source credibility
+  {
+    url: "https://pesacheck.org/tagged/burkina-faso",
+    mediaType: MediaType.image,
+    imageGridIndex: 0,
+    services: [MediaServices.metadata, MediaServices.magnifier, MediaServices.forensic, MediaServices.ocr],
+    namedEntities: {
+      Person: [
+        "Citizens"
+      ]
+    },
+    domainAnalyses: {
+      factChecker: ["Duke Reporters' Lab"]
+    },
+    extractedURLAnalyses: {
+        "https://pesacheck.org/about": LinkResult.positive,
+    },
+    credibilitySignals: {
+      topic: [
+        "Law and Justice System",
+        "Crime and Punishment", // Not highlighted
+        "Politics"
+      ],
+      genre: ["Satire"], // I don't think so...
+      persuasionTechniques: [
+        "Appeal to Authority",
+        "Loaded language",
+      ],
+      subjectivity: ["None detected"]
+    }
+  },
+  // All three source types
+  {
+    url: "https://demagog.cz/vyrok/23170",
+    mediaType: MediaType.image,
+    imageGridIndex: 0,
+    services: [MediaServices.metadata, MediaServices.magnifier, MediaServices.forensic, MediaServices.ocr],
+    namedEntities: {
+      // There should definitely be some named entities in here, e.g. Andreje Babiše, but maybe the service doesn't
+      // work in Czech?
+    },
+    domainAnalyses: {
+      factChecker: ["Meta", "Duke Reporters' Lab"]
+    },
+    extractedURLAnalyses: {
+        "https://twitter.com/DemagogCZ": LinkResult.positive,
+        "https://www.facebook.com/journalismproject/programs/third-party-fact-checking": LinkResult.mention,
+        "https://drive.google.com/drive/folders/1J6nkGqAan4B5tet7dG5I9rRxHGyUwHGq": LinkResult.negative,
+        "https://web.archive.org/web/20230323215342/https:/www.ustavnysud.sk/c/document_library/get_file?uuid=67bda80c-2905-43c2-a802-7796b0c14c31&groupId=10182": LinkResult.negative
+    },
+    credibilitySignals: {
+      topic: [
+        "Law and Justice System",
+        "Crime and Punishment", // Not highlighted
+        "Politics"
+      ],
+      genre: ["Objective reporting"],
+      persuasionTechniques: [
+        "Appeal to Authority",
+        "Appeal to Hypocrisy",
+        "Appeal to time",
+        "Appeal to values",
+        "Conversation killer",
+        "Doubt",
+        "Guilt by association",
+        "Loaded language",
+        "Name calling or labeling",
+        "Questioning the reputation",
+      ],
+      // TODO: Maybe try to test _which_ sentences are subjective?
+      subjectivity: ["Subjective sentences detected (4/60)"]
```

Comment:
This will change to remove the number of sentences 
 https://github.com/AFP-Medialab/InVID-Translations/blob/df4c03af3ec6f5d27511ce3e7c91302a25c7ed8f/components/NavItems/tools/Assistant.tsv#L180